go.mod

```diff
@@ -49,6 +48,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	github.com/transip/gotransip v5.8.2+incompatible
 	github.com/vinyldns/go-vinyldns v0.0.0-20190611170422-7119fe55ed92
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200401174654-e694b7bb0875
```

Comment:
Concerned about this weird version number? Blame it on go modules :(.

`go.etcd.io/etcd` is using  `go.mod` and go modules won't allow releases of `v3.x` unless it follows the semantic import versioning.

That version above still corresponds to the commit hash of `v3.4.7`, the release.